private/buf/cmd/buf/command/breaking/breaking.go

```diff
@@ -219,6 +219,11 @@ func run(
 		if flags.ExcludeImports {
 			breakingOptions = append(breakingOptions, bufcheck.BreakingWithExcludeImports())
 		}
+		if len(imageWithConfig.PluginConfigs()) > 0 {
+			if err := bufcli.CheckBetaPluginEnabled(container); err != nil {
+				return fmt.Errorf("plugin configs are set but custom breaking plugins are not enabled: %w", err)
+			}
```

Comment:
We may need to surface a more detailed error message, something like:

```
There are plugins set for breaking change detection, however this feature is in beta and not yet officially released. In order to use plugins, set the environment variable BUF_BETA_PLUGINS_ENABLED=1.
```

In which case, you may need to change the way the env var is checked (the function returns a `bool` and then you construct a detailed error here).

Reply:
I think [6c614dc](https://github.com/bufbuild/buf/pull/3288/commits/6c614dc6cdc5b10cd21f4b1ab478a034912825df) addresses it, but it takes a slightly different approach